src/App.js

```diff
@@ -62,26 +63,31 @@ function App() {
 	}
 
 	return (
-		<BrowserRouter>
-			<Routes>
-				<Route
-					path='/'
-					element={
-						<>
-							{isLoading ? (
-								<p>Loading...</p>
-							) : (
-								<div>
-									<h2> Todo List:</h2>
-									<TodoList todoList={todoList} onRemoveTodo={removeTodo} />
-								</div>
-							)}
-							<AddTodoForm onAddTodo={addTodo} />
-						</>
-					}></Route>
-				<Route path='/new' element={<h1>New Todo List</h1>}></Route>
-			</Routes>
-		</BrowserRouter>
+		<div className={style.container}>
```

Comment:
This `container` prop is very generic. Not too big of a deal in an app of this size but avoid generic naming throughout your app. Especially now that your applications will be larger.